terraform/modules/environment/main.tf

```diff
@@ -50,8 +50,23 @@ module "eks" {
   worker_ami           = var.eks_worker_ami # EKS-Optimized AMI for your region: https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html
 }
 
-data "aws_iam_user" "ci_user" {
-  user_name = "ci-user" # Should have been created in the bootstrap process
+# ci user for ECR
+resource "aws_iam_user" "ci_user" {
+  name = "${var.project}-ci-user"
+}
+
+# Create a keypair to be used by CI systems
+resource "aws_iam_access_key" "ci_user" {
+  user    = aws_iam_user.ci_user.name
+}
+
+# Add the keys to AWS secrets manager
+module "ci_user_keys" {
```

Comment:
Hmm we could have left this in the secrets section. It still might be beneficial to have that for secrets like this that we just want to put into secrets manager and forget about. I know it's another step and that's annoying, but we should be careful how many secrets get put into the TF state.